src/styles/tickets/frontend.pcss

```diff
@@ -3,308 +3,339 @@
  *
  * @since  0.3.0-alpha
  */
-.tribe-block__tickets {
+.tribe-common.tribe-block__tickets {
+	border: 1px solid var(--color-border-secondary);
+	border-radius: 4px;
 	margin-top: 30px;
 	margin-bottom: 30px;
+	padding: 1rem;
 }
 
-.tribe-block__tickets__item {
-	display: flex;
-	flex-wrap: wrap;
-	background: #fff;
-	border: 1px solid #dfdfdf;
-	border-bottom-width: 0;
+ .tribe-common {
 
-	&:last-of-type {
-		border-bottom-width: 1px;
-	}
-}
+	 .tribe-block__tickets__title {
+		 font-weight: var(--font-weight-regular);
+	 }
 
-.tribe-block__tickets__item__icon {
-	background: #f9f9f9;
-	flex: 1 1 70px;
-	padding: 25px 10px;
-	text-align: center;
+	.tribe-block__tickets__item {
+		display: flex;
+		flex-flow: row wrap;
+		background: var(--color-background);
+		border-bottom: 1px solid var(--color-border-secondary);
+		border-top: 1px solid var(--color-border-secondary);
+		padding: .875em 0;
 
-	.tribe-svg-icon-ticket {
-		display: block;
-		width: 100%;
-		height: auto;
+		&:last-of-type {
+			border-bottom-width: 1px;
+		}
 	}
-}
 
-.tribe-block__tickets__item__content {
-	flex: 10 5 400px;
-	padding: 25px;
-}
+	.tribe-block__tickets__item__icon {
+		background: #f9f9f9;
+		flex: 1 1 70px;
+		text-align: center;
 
-.tribe-block__tickets__item__content__title {
-	display: block;
-	font-weight: bold;
-	font-size: 1.1em;
-}
+		.tribe-svg-icon-ticket {
+			display: block;
+			width: 100%;
+			height: auto;
+		}
+	}
 
-.tribe-block__tickets__item__content__description {
-	display: block;
-	font-size: 0.8em;
-	margin-top: 5px;
-}
+	.tribe-block__tickets__item__content {
+		align-items: baseline;
+		display: flex;
+		flex: 1 1 calc(100% - 90px);
+		flex-wrap: wrap;
+	}
 
-.tribe-block__tickets__item__extra {
-	flex: 1 1 200px;
-	padding: 25px 10px;
-	text-align: center;
-}
+	.tribe-block__tickets__item__content__title {
+		display: block;
+		flex: 1 1 100%;
+		font-weight: bold;
+		order: 1;
 
-.tribe-block__tickets__item__extra__price {
-	font-size: 1.1em;
-	font-weight: bold;
-}
+		@media(--viewport-medium) {
+			flex: 0 0 80%;
+		}
+	}
 
-.tribe-block__tickets__item__extra__available {
-	font-size: 0.8em;
-	margin-top: 5px;
-}
+	.tribe-block__tickets__item__content__description {
+		display: block;
+		flex: 1 1 auto;
+		margin-left: .5em;
```

Comment:
em, also let's use spacer vars.

Reply:
Yeah, these are going away - ignore this file.